opensrp-pnc/src/main/java/org/smartregister/pnc/utils/PncDbConstants.java

```diff
@@ -54,12 +53,39 @@
             String EVENT_DATE = "event_date";
             String CREATED_AT = "created_at";
         }
+
+        interface PncBaby {
+            String MOTHER_BASE_ENTITY_ID = "mother_base_entity_id";
+            String FIRST_NAME = "first_name";
+            String LAST_NAME = "last_name";
+            String DOB = "dob";
+            String GENDER = "gender";
+            String DISCHARGED_ALIVE = "discharged_alive";
+            String WEIGHT = "weight";
+            String HEIGHT = "height";
+            String APGAR = "apgar";
+            String FIRST_CRY = "first_cry";
+            String COMPLICATIONS = "complications";
+            String COMPLICATIONS_OTHER = "complications_other";
+            String NVP_ADMINISTRATION = "nvp_administration";
+            String BF_FIRST_HOUR = "bf_first_hour";
+            String INTERVENTION_REFERRAL_LOCATION = "intervention_referral_location";
+            String INTERVENTION_SPECIFY = "intervention_specify";
+            String CARE_MGT = "care_mgt";
+        }
+
+        interface PncStillBorn {
+            String MOTHER_BASE_ENTITY_ID = "mother_base_entity_id";
+            String STILLBIRTH__CONDITION = "stillbirth_condition";
+        }
     }
 
     interface Table {
         String EC_CLIENT = "ec_client";
         String PNC_DETAILS = "pnc_details";
         String PNC_REGISTRATION_DETAILS = "pnc_registration_details";
         String PNC_OUTCOME_FORM = "pnc_outcome_form";
+        String PNC_BABY = "pnc_baby";
+        String PNC_STILL_BORN = "pnc_still_born";
```

Comment:
you can remove this

Reply:
PNC_STILL_BORN removed. Whereas PNC_BABY used in ChildRepository